static/panes/compiler.js

```diff
@@ -550,6 +550,7 @@ Compiler.prototype.compile = function (bypassCache, newTools) {
                 pass: this.gccDumpPassSelected,
                 treeDump: this.treeDumpEnabled,
                 rtlDump: this.rtlDumpEnabled,
+                dumpFlags: this.dumpFlags,
```

Comment:
I would rather we simply pass the bools for each option here and let the server side handle the actual command line arguments

Reply:
I would rather have an enum flag instead of lots of booleans.